Gruntfile.js

```diff
@@ -188,7 +185,19 @@ module.exports = function (grunt) {
     // Exorcise, Uglify, Concat are used to create a minimized freedom.js with
     // correct sourcemap. Uglify needs an explicit 'sourceMapIn' argument,
     // requiring that exorcise be used before hand. Concat is able to properly
-    // attach a banner while maintaining the correct source-map offsets.
+    // attach a banner while maintaining the correct source-map offsets.    
+    // The replace tasks removes a charset declaration that breaks concat -
+    // it should be removed once concat copes better.
+    replace: {
```

Comment:
Just making sure that break points line up and can be set in the source-map
files appropriately, or if they end up breaking a line off.  if things seem
good, then this sounds good.

On Mon, Jul 13, 2015 at 11:43 AM, soycode <notifications@github.com> wrote:

> In Gruntfile.js
> <https://github.com/freedomjs/freedom/pull/286#discussion_r34496656>:
>
> > @@ -188,7 +185,19 @@ module.exports = function (grunt) {
> >      // Exorcise, Uglify, Concat are used to create a minimized freedom.js with
> >      // correct sourcemap. Uglify needs an explicit 'sourceMapIn' argument,
> >      // requiring that exorcise be used before hand. Concat is able to properly
> > -    // attach a banner while maintaining the correct source-map offsets.
> > +    // attach a banner while maintaining the correct source-map offsets.
> > +    // The replace tasks removes a charset declaration that breaks concat -
> > +    // it should be removed once concat copes better.
> > +    replace: {
>
> Loading up the generated freedom.min.js(.map) in Chrome and playing with
> it a bit, it looks like line numbers correspond appropriately. Not sure if
> you were envisioning/intending some other check, let me know - thanks!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/freedomjs/freedom/pull/286/files#r34496656>.
>